server/db/models/user.js

```diff
@@ -33,6 +33,10 @@ export default (sequelize, DataTypes) => {
       profilePic: {
         type: DataTypes.STRING,
         allowNull: true
+      },
+      role: {
+        type: DataTypes.STRING,
+        allowNull: false
```

Comment:
You can also set a default value so it doesn't have to stated everything

Reply:
true, default value should be user. But it is also set during signup. I just thought to leave it like that.